pkg/operation/seed/seed.go

```diff
@@ -115,13 +222,89 @@ func BootstrapCluster(seed *Seed, k8sGardenClient kubernetes.Client, secrets map
 	if err != nil {
 		return err
 	}
+	elasticsearchVersion, err := imageVector.FindImage("elasticsearch-oss", k8sGardenClient.Version())
+	if err != nil {
+		return err
+	}
+	fluentdEsVersion, err := imageVector.FindImage("fluentd-es", k8sGardenClient.Version())
+	if err != nil {
+		return err
+	}
+	curatorEsVersion, err := imageVector.FindImage("curator-es", k8sGardenClient.Version())
+	if err != nil {
+		return err
+	}
+	kibanaVersion, err := imageVector.FindImage("kibana-oss", k8sGardenClient.Version())
+	if err != nil {
+		return err
+	}
+	alpineVersion, err := imageVector.FindImage("alpine", k8sGardenClient.Version())
+	if err != nil {
+		return err
+	}
+
+	var (
+		basicAuth  string
+		kibanaHost string
+		replicas   int
+	)
+
+	loggingEnabled := features.ControllerFeatureGate.Enabled(features.Logging)
+
+	if loggingEnabled {
+		deployedSecrets, err := k8sSeedClient.ListSecrets(common.GardenNamespace, metav1.ListOptions{})
+		if err != nil {
+			return err
+		}
+
+		for _, secret := range deployedSecrets.Items {
+			secretObj := secret
+			deployedSecretsMap[secret.ObjectMeta.Name] = &secretObj
+		}
+
+		// currently the generated certificates are only used by the kibana so they are all disabled/enabled when the logging feature is disabled/enabled
+		err = deployCertificates(seed, k8sSeedClient, deployedSecretsMap)
+		if err != nil {
+			return err
+		}
+
+		username, password, err := generateKibanaIngressCredentials(seed, k8sSeedClient, deployedSecretsMap)
+		if err != nil {
+			return err
+		}
+
+		basicAuth = utils.CreateSHA1Secret(username, password)
+		kibanaHost = seed.GetIngressFQDN("k", seed.Info.Name, "garden")
```

Comment:
Why not simply `k`?

Reply:
Was following the pattern <initial>.<shoot>.<project>.<ingress-domain>

Reply:
Which pattern? The kibana ingress is unique for this seed, hence, let's at least remove the seed name and use `k.garden.<ingressDomain>`. The `<ingressDomain>` already identifies the seed.